.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+version: 2
```

Comment:
I added this because when I tested the workflows locally, I got warnings about Node 16 workflows. You can remove if you like.